app/containers/MessageActions/index.js

```diff
@@ -10,12 +10,13 @@ import I18n from '../../i18n';
 import log, { logEvent } from '../../utils/log';
 import Navigation from '../../lib/Navigation';
 import { getMessageTranslation } from '../message/utils';
-import { LISTENER } from '../Toast';
+import { showToast } from '../Toast';
 import EventEmitter from '../../utils/events';
 import { showConfirmationAlert } from '../../utils/info';
 import { useActionSheet } from '../ActionSheet';
 import Header, { HEADER_HEIGHT } from './Header';
 import events from '../../utils/log/events';
+import { show } from 'react-native-bootsplash';
```

Comment:
unnecessary import

Reply:
Yep, any unused import should be removed, check the lint stage of our CI pipeline:
https://app.circleci.com/pipelines/github/RocketChat/Rocket.Chat.ReactNative/1914/workflows/4920f968-6c8b-4150-8f8b-9055b31fea92/jobs/20908
This is a required stage to a PR be merged